tests/acceptance/learnergroup-bulk-assign-test.js

```diff
@@ -159,18 +171,20 @@ module('Acceptance | learner group bulk assign', function(hooks) {
     let users = [
       ['j', 'johnson', '1234567890', '123Test'],
       ['jackson', 'j', '12345'],
-      ['', 'johnson', '12345', '123Test'],
-      ['Magick', '', '12345'],
+      ['', 'johnson', '666666', '123Test'],
```

Comment:
👿 🔥

Reply:
@jrjohnson jebediah johnson, heretic. it is known!